### PR TITLE
fix: hamburger menu offset

### PR DESCRIPTION
**Pull request by sashuk**
[FX-4200]

### Description

This pull request fixes the hamburger menu offset and the hamburger button is vertically centered.

Design https://www.figma.com/file/5SCTOPrCDcHuk5We091GBn/Product-Library?type=design&node-id=9938-40726&mode=design&t=brwuhESCM5CsNynL-0

### How to test

- [Temploy](https://picasso.toptal.net/FX-NULL-fix-compact-dropdown-menu)
- Compare with https://picasso.toptal.net/?path=/story/layout-page--page side-by-side by opening the hamburger menu

### Screenshots

Hamburger button position before

Screenshot 2023-07-26 at 11 53 50

Hamburger button position after

Screenshot 2023-07-26 at 11 53 57

Hamburger button position in design

Screenshot 2023-07-26 at 11 54 03

---

Please see the side-by-side comparison of hamburger button position and menu (in `master` / in this pull request)

https://github.com/toptal/picasso/assets/1390758/81442492-a25c-4b75-b9c0-85aaddfcd606

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [N/A] Annotate all `props` in component with documentation
- [N/A] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [N/A] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [N/A] Covered with tests

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4200]: https://toptal-core.atlassian.net/browse/FX-4200?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/picasso/src/PageHamburger/styles.ts [hunk 1]
```diff
@@ -8,7 +8,6 @@ export default ({ palette }: Theme) => {
 
   return createStyles({
     root: {
-      display: 'block',
```
Comment (sashuk) [inline packages/picasso/src/PageHamburger/styles.ts:11]: Needed for hamburger button to be properly centered according to https://www.figma.com/file/5SCTOPrCDcHuk5We091GBn/Product-Library?type=design&node-id=9938-40726&mode=design&t=brwuhESCM5CsNynL-0

File: packages/picasso/src/PageHamburger/PageHamburger.tsx [hunk 2]
```diff
@@ -23,6 +24,8 @@ const PageHamburger = ({ id, 'data-testid': dataTestId }: Props) => {
   const [showContent, setShowContent] = useState<boolean>(false)
   const classes = useStyles()
 
+  const nonXlLayout = useBreakpoint(['xs', 'sm', 'md', 'lg'])
```
[the remark this answers is not in the document]
Comment (sashuk) [reply packages/picasso/src/PageHamburger/PageHamburger.tsx]: @TomasSlama Addressed, thank you!

**Review by TomasSlama — APPROVED** [review]
thank you 🙇

**Review by augustobmoura — APPROVED** [review]
Nice catch! :+1:

**Pull request merged**